app/test_main.py

```diff
@@ -1,3 +1,33 @@
+import pytest
 from app.main import get_human_age
+from typing import Any
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17]),
+    (-1, 10, [0, 0]),
+    (10, -1, [0, 0]),
+    (0, 10, [0, 0]),
+    (10, 0, [0, 0]),
+    (1000, 1000, [246, 197]),
+    ("cat", 10, TypeError),
+    (10, "dog", TypeError),
```

Comment:
Add test cases for errors for inputs other than strings, e.g. lists, sets, tuples, etc.